games/s.yaml

```diff
@@ -162,6 +162,20 @@
   license: [As-is]
   remakes: [Shining Soul]
 
+- name: Shockolate
+  updated: 2018-06-15
+  repo: https://github.com/Interrupt/systemshock
+  development: active
+  status: playable
+  lang: [C]
+  framework: [SDL2]
+  license: GPL3
+  content: commercial
```

Comment:
I am confused what `content` means. I assume it means game data as non-code files (graphics, music, etc). But does it describe what's **inside** the repo? Or what needs to be **supplied** (for remakes)? In Shockolate, there's no game content, because you need to supply original commercial files. Is that field value correct? (Also, could you explain this better in readme? Thanks).

Reply:
See #373. It would be commercial

Reply:
Ok. Anything else before this can be merged?